src/pages/Authentication.jsx

```diff
@@ -1,24 +1,26 @@
 import React from 'react';
 
-import Navbar from '../components/presentationals/Navbar.jsx';
-import AuthenticationIllustration from '../components/presentationals/AuthenticationIllustration.jsx';
-import AuthenticationCard from '../components/presentationals/AuthenticationCard.jsx';
-import './Authentication.scss';
+import Navbar from '../components/presentationals/Navbar/Navbar';
+import AuthenticationIllustration from '../components/presentationals/AuthenticationIllustration/AuthenticationIllustration';
+import AuthenticationCard from '../components/presentationals/AuthenticationCard/AuthenticationCard';
+import '../pages/Authentication.scss';
 
-const AuthenticationLayout = () => {
+const AuthenticationLayout = testNavbars => {
```

Comment:
`testNavbars` is not a react props. 
`const  AuthenticationLayout = { profileUrl } => {`
`<Navbar props={{profileUrl}} />

Reply:
Done